Kitodo/src/main/java/org/kitodo/production/forms/ProcessForm.java

```diff
@@ -99,6 +102,7 @@ public class ProcessForm extends TemplateBaseForm {
     private static final String CREATE_PROCESS_PATH = "/pages/processFromTemplate.jsf?faces-redirect=true";
     private static final String PROCESS_TABLE_VIEW_ID = "/pages/processes.xhtml";
     private static final String PROCESS_TABLE_ID = "processesTabView:processesForm:processesTable";
+    private final Map<Integer, Boolean> assignedProcesses = new HashMap<>();
```

Comment:
Map can be simplified from `<Integer, Boolean>` to `<int, boolean>` as `null` values are invalid for process ids and true/false (is process in project or not).